src/integrators/controllers.jl

```diff
@@ -308,11 +337,6 @@ end
     return dt_factor
 end
 
-# checks whether the controller should accept a step based on the error estimate
-@inline function accept_step_controller(integrator, controller::AbstractController)
-    return integrator.EEst <= 1
-end
```

Comment:
This fallback should be implemented again for the other controllers (like PI), shouldn't it?

Reply:
I just move this part of code at the top of the file because I find it clearer ?

Reply:
Ok, sorry. I just saw that this method was deleted in the big diff on GitHub